scripts/js/footer.js

```diff
@@ -7,31 +7,33 @@
 
 /* global utils:false, moment:false */
 
-var _isLoginPage = false;
+"use strict";
+
+globalThis._isLoginPage = false;
```

Comment:
This should have been a global variable in the first place since we manipulate it from other JS files.